tendrl/notifier/notification/__init__.py

```diff
@@ -51,11 +53,11 @@ def dispatch_notification(self, msg):
         raise NotImplementedError()
 
 
-class NotificationPluginManager(object):
+class NotificationPluginManager(gevent.greenlet.Greenlet):
```

Comment:
no it is working now, now i have only NotificationPluginManager gevent in notifier module